bin/packages/build.js

```diff
@@ -203,8 +207,10 @@ function buildPackage( packagePath ) {
 	// Build js files individually.
 	jsFiles.forEach( ( file ) => buildJsFile( file, true ) );
 
-	// Build entire package scss.
-	buildPackageScss( packagePath );
+	const scssFiles = glob.sync( `${ srcDir }/*.scss` );
```

Comment:
So if I understand properly, this considers all root level files in as entry points. I personally don't like this convention because in some packages a `src/style.scss` may import and `src/something.scss` and we'd still have a unique entry point.

I think I'd prefer having the current behavior: the  `src/style.scss` file convention as a fallback and explicit entry points (maybe a custom property in the `package.json` in case there are multiple)

That said, I can be ok with the current behavior temporarily, I'd like if we document it somewhere though.

Last thing, I think this breaks "watching" sass files, we probably have to rebuild all the files and not only the `style.scss` one if a sass file changes.

Reply:
Personally, I would go further and glob for all `scss` files, the same way we do it for JS files. The next step would be a bit different. Instead of transpiling all of them as it happens for JS, we would group them based on the file name and produce one `.css`, e.g. all `style.scss` files from one package would get merged into `build-style/style.css`.

```js
// Exclude deceitful source-like files, such as editor swap files.
const isSourceFile = ( filename ) => {
	return /.\.(js|scss)$/.test( filename );
};
```

> Last thing, I think this breaks "watching" sass files, we probably have to rebuild all the files and not only the `style.scss` one if a sass file changes.
```
execSync( `${ BUILD_CMD } ${ files.join( ' ' ) }`, { stdio: [ 0, 1, 2 ] } );
```
Yes, correct. We need to improve this one somehow. It probably would make sense to implement the grouping I described which would ultimately solve the latter.